actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -577,7 +577,7 @@ public void setDeactivationTimeout(long deactivationTimeoutMs)
         logger.info("Starting Stage...");
         extensions = new ArrayList<>(extensions);
         startCalled = true;
-        if (state != null)
+        if (state != null && state != NodeCapabilities.NodeState.STOPPED)
```

Comment:
might also want to introduce a lock to prevent concurrent start / stop invocations.